examples/large_models/inferentia2/llama2/model-config.yaml

```diff
@@ -0,0 +1,17 @@
+minWorkers: 1
+maxWorkers: 1
+maxBatchDelay: 100
+responseTimeout: 10800
+batchSize: 16
+
+handler:
+    amp: "bf16"
+    tp_degree: 6
+    max_length: 100
+
+micro_batching:
```

Comment:
Add comments describing intuition of selecting good values based on size of the inf2 instance types

Reply:
Added details about batch size and micro batch size to the Readme.